webpack.config.development.js

```diff
@@ -1,14 +1,13 @@
 'use strict';
 
+var _ = require('lodash');
 var webpack = require('webpack');
 var baseConfig = require('./webpack.config.base');
 
-var config = Object.create(baseConfig);
-config.plugins = [
-  new webpack.optimize.OccurenceOrderPlugin(),
-  new webpack.DefinePlugin({
-    'process.env.NODE_ENV': JSON.stringify('development')
-  })
-];
-
-module.exports = config;
+module.exports = _.merge({}, baseConfig, {
+  plugins: _.union(baseConfig.plugins, [
```

Comment:
Just curious, but why use 'union' here? The plugins are all new objects, seems like basic concat would suffice.